services/follower.service.js

```diff
@@ -0,0 +1,26 @@
+import { Follower } from '../db/models/follower.js';
+import { EntityService } from './entity.service.js';
+
+export class FollowerService extends EntityService {
+  constructor() {
+    super(Follower);
+  }
+
+  async follow(payload) {
+    if (payload.userId === payload.accountId) {
+      throw new Error(400, 'You cannot follow yourself.');
+    }
+
+    const result = await Follower.query()
```

Comment:
What about using just return?

```js
return Follower.query()
      .insert(payload)
      .onConflict(['userId', 'accountId'])
      .merge();
```

Reply:
btw, EntityService has getter queryBuilder to avoid using Follower.query()

u can replace it with this.queryBuilder...